src/utils.ts

```diff
@@ -210,3 +210,6 @@ export const isTextInputLikeElement = (el: HTMLElement) => {
     el.isContentEditable
   );
 };
+
+export const isTabLikeElement = (el: HTMLElement) =>
+  el.getAttribute('role') === 'tab';
```

Comment:
An interesting fact is that `@testing-library/dom` cheks all space-delimited roles:
https://github.com/testing-library/dom-testing-library/blob/fbbb29a6d9655d41bc8f91d49dc64326f588c0d6/src/queries/role.js#L107-L112

Should we use the same logic for that? 🤔 

nit: this probably should be called `isTabElement` or `hasTabRole`

Reply:
We'll probably have the very same issue with arrows triggering canvas interactions when navigating through menus. Damn, handling all the cases here is hard! 😬